java/code/src/com/redhat/rhn/domain/dto/EndpointInfo.java

```diff
@@ -0,0 +1,96 @@
+/**
+ *
```

Comment:
```suggestion
 * Copyright (c) 2021 SUSE LLC
 *
```
The copyright header is missing for all new classes in this patch, wouldn't checkstyle usually remind us about that?

Reply:
Please add the copyright line everywhere. without it we cannot let this pass. When the checkstyle failed, we need to check why. But it is not an option to remove it.

Reply:
Done